src/app/terra-components.module.ts

```diff
@@ -370,6 +375,8 @@ function createCompiler(compilerFactory:CompilerFactory):Compiler
         TerraDynamicSwitchComponent,
         TerraCategoryPickerComponent,
         TerraTwoColumnsContainerComponent,
+        TerraBreadcrumbsComponent,
+        TerraTwoColumnsContainerDirective,
```

Comment:
You need to add an export in the index.ts file (it is directly in the src folder)